web/app/i18n/fr.json

```diff
@@ -0,0 +1,321 @@
+        "SIGN_IN": "Se connecter",
+        "USERNAME": "Nom d'utilisateur"
+    },
+    "MANAGEMENT": {
+        "ACCOUNT_INFORMATION": "Renseignements concernant le compte",
+        "ADMINISTRATOR?": "Administrateur?",
+        "ARCHIVE_NOTE": "L'autre enregistrement sera conservé pour l'avenir.",
+        "AUDIT_LOGS": "Journaux d'audit mensuels",
+        "EMAIL": "Email",
+        "GROUPS": "Groupes",
+        "KEEP_BOTH_RECORDS": "Conservez tous les deux enregistrements uniques",
+        "POTENTIAL_DUPLICATES": "Enregistrements en double potentiels",
+        "REALLY_ARCHIVE": "Êtes-vous sûr de vouloir utiliser cet enregistrement et d’archiver les données de l’autre enregistrement?",
+        "RESOLVE": "Résoudre",
+        "RESOLVE_DUPLICATE_ENTRIES": "Résoudre les enregistrements en double",
+        "TOKEN": "Token",
+        "USE_THIS_RECORD": "Utilisez cet enregistrement",
+        "USER_ID": "Identifiant d'utilisateur"
+    },
+    "MAP": {
+        "BLACKSPOT": "Point noir",
+        "BLACKSPOTS": "Points noirs",
+        "CDB_ATTRIBUTION": "!&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors, &copy; <a href='https://cartodb.com/attributions'>CartoDB</a>",
+        "ESRI_ATTRIBUTION": "!Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community",
+        "HEATMAP": "Carte thermique",
+        "NUM_SEVERE": "Nombre d'incidents graves",
+        "SEVERITY_SCORE": "Score de gravité",
+        "STREETS": "Rues",
+        "SATELLITE": "Satellite"
+    },
+    "MONTH": {
+        "JANUARY": "janvier",
+        "FEBRUARY": "janvier",
+        "MARCH": "mars",
+        "APRIL": "avril",
+        "MAY": "mai",
+        "JUNE": "juin",
+        "JULY": "juillet",
+        "AUGUST": "août",
+        "SEPTEMBER": "septembre",
+        "OCTOBER": "octobre",
+        "NOVEMBER": "novembre",
+        "DECEMBER": "décembre"
+    },
+    "NAV": {
+        "ACCOUNT": "Compte",
+        "ADD_A_RECORD": "Ajouter un enregistrement",
+        "ALL": "Tout",
+        "CUSTOM_REPORT": "Rapport personnalisé",
+        "DASHBOARD": "Tableau de bord",
+        "DOWNLOAD_AUDIT_LOGS": "Télécharger les enregistrements d'audit",
+        "DUPLICATES": "Doublons potentiels",
+        "EDIT_A_RECORD": "Editer un enregistrement",
+        "LOG_OUT": "Déconnexion",
+        "MAP": "Carte",
+        "MANAGE_DUPLICATE_RECORDS": "Gérer les enregistrements en double",
+        "MY_ACCOUNT": "Mon compte",
+        "RECORD_DETAILS": "Détails des enregistrements",
+        "RECORD_LIST": "Liste des enregistrements",
+        "ENFORCER_ASSIGNMENTS": "Mise en vigueur"
+    },
+    "RECORD": {
+        "BUTTON_ADD_ROW_TITLE": "Ajouter \\{\\{0\\}\\}",
+        "BUTTON_COLLAPSE": "Effondrer",
+        "BUTTON_DELETE_ROW_TITLE": "Supprimer \\{\\{0\\}\\}",
+        "BUTTON_EXPAND": "Elargir",
+        "CREATED": "Créé",
+        "MODIFIED": "!Modified",
```

Comment:
There are 6 untranslated strings here.

Reply:
Same reason the one above was left out (created after the translation spreadsheet).  Requested new translations from WB.